website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -203,11 +203,15 @@ status:
       instanceMatchCriteria: targeted
       instanceType: g6.48xlarge
       ownerID: "012345678901"
+      reservationType: capacity-block
+      state: expiring
```

Comment:
Seems to be missing in some other paths e.g. `website/content/en/docs/concepts`.

Reply:
This is intentional, these fields were added to support capacity blocks and will only be present in 1.6+.